tests/wdio/secondaryScroll-spec.js

```diff
@@ -0,0 +1,13 @@
+Terra.describeViewports('secondary scroll multiple files', ['huge'], () => {
+  it('should scroll to the correct file path based on navigation', () => {
+    browser.url('/secondary-scroll/terra-dev-site/multiple-files/example-2');
+    Terra.validates.element({ selector: '#root' });
```

Comment:
Should we be clicking a new item in the navigation pane and then returning to the previous content to show the scroll is working?

Reply:
Returning to the previous content like clicking the back button? Is there a way to do that in wdio tests?